app/views/events/_future.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="m-5">
+<% if Event.future.nil? %>
```

Comment:
You should make use of a variable instead. Check the comment of the event index controller.